submission/persistence-mongo/src/main/kotlin/ac/uk/ebi/biostd/persistence/doc/migrations/DatabaseChangeLog.kt

```diff
@@ -197,3 +205,11 @@ suspend fun ReactiveMongoOperations.ensureStatsIndexes() {
         ensureIndex(backgroundIndex().on(SUB_ACC_NO, ASC)).awaitSingleOrNull()
     }
 }
+
+/**
+ * Rename the field userId to owner in the submission_drafts collection
+ */
+suspend fun ReactiveMongoOperations.renameDraftOwnerField() {
```

Comment:
I think we should either add this and remove it (add a todo) or adding manually, as everydeploy even if there is not real update every draft document will be try to be updated